src/prefect/task_worker.py

```diff
@@ -76,28 +84,30 @@ class TaskWorker:
 
     def __init__(
         self,
-        *tasks: Task,
+        *tasks: Task[P, R],
         limit: Optional[int] = 10,
     ):
-        self.tasks = []
+        self.tasks: list[Task[Any, Any]] = []
```

Comment:
```suggestion
        self.tasks: list["Task[..., Any]"] = []
```

Quoted because Python 3.9 will croak otherwise.